docs/extensions/datacollector.md

```diff
@@ -93,12 +93,14 @@ DataCollectors can choose to subscribe to any of the four events exposed by `Dat
 2. TestSessionEnd : Raised when test execution session ends.
 3. TestCaseStart : Raised when test case execution starts.
 4. TestCaseEnd : Raised when test case execution ends.
+5. TestHostInitialized : Raised when test host process has been initialized.
```

Comment:
From which version onwards will this be available? Can you mention that, please.

Reply:
15.7 blame data collector has this feature and Microsoft.Testplatform.ObjectModel this as well.